src/common/POLFields/VendorFields/FieldVendorRefNumber.js

```diff
@@ -1,21 +1,40 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { FormattedMessage } from 'react-intl';
 import { Field } from 'redux-form';
 
 import { TextField } from '@folio/stripes/components';
 
 import { requiredRefNumberType } from '../../../components/Utils/Validate';
 
-const FieldVendorRefNumber = () => {
+const FieldVendorRefNumber = ({ dispatch, change, required }) => {
+  const onChange = (_, value) => {
+    dispatch(change('vendorDetail.refNumber', value));
```

Comment:
you keep putting dispatches of actions, but it should be working via redux without them explicit

Reply:
Fixed. It works